Projects/Delivery/Db/src/test/java/com/ilmlf/delivery/db/DbTest.java

```diff
@@ -0,0 +1,52 @@
+package com.ilmlf.delivery.db;
```

Comment:
Did this end up working? I wasn't sure if the plan was still to keep the snapshots or move entirely to cdk diff

Reply:
Yes it does work now !